jdi-light/src/main/java/com/epam/jdi/light/asserts/core/DataListAssert.java

```diff
@@ -110,7 +109,7 @@
     @JDIAction("Assert that '{name}' elements [{0}] are displayed")
     public DataListAssert<T, D> displayed(String... names) {
         for (String name : names)
-            jdiAssert(element.get(name).isDisplayed() ? name + "displayed" : "hidden", Matchers.is(name + "displayed"));
+            jdiAssert(element().get(name).isDisplayed() ? name + "displayed" : "hidden", Matchers.is(name + "displayed"));
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unnecessary use of fully qualified name 'Matchers.is' due to existing static import 'org.hamcrest.Matchers.*'](https://app.codacy.com/manual/42_/jdi-light/pullRequest?prid=5576697)